vnext/local-cli/runWindows/runWindows.js

```diff
@@ -51,8 +52,9 @@ async function runWindows(config, args, options) {
   const slnFile = options.sln || build.getSolutionFile(options);
 
   if (options.autolink) {
-    autolink.updateAutoLink(verbose);
+    await autolink.func(null, null, {logging: options.logging});
```

Comment:
names for the params? if we pass an options param, maybe make the 1st two params be part of that dictionary so you can name what those nulls are more easily

Reply:
The structure for the command is the format that react-native subcommands use. I suppose I could pass in the values that were passed into `run-windows`. I don't use them within auto-linking.

Reply:
Fixed.